app/code/Magento/Sales/Model/Order/Invoice.php

```diff
@@ -96,9 +96,9 @@ class Invoice extends AbstractModel implements EntityInterface, InvoiceInterface
     protected $_invoiceConfig;
 
     /**
-     * @var \Magento\Sales\Model\OrderFactory
+     * @var \Magento\Sales\Api\OrderRepositoryInterface
      */
-    protected $_orderFactory;
```

Comment:
Removing `$_orderFactory` may break clients already relying on this property. It's better to deprecate it.